app/controllers/api/v1/server_tags_controller.rb

```diff
@@ -0,0 +1,34 @@
+# BigBlueButton open source conferencing system - http://www.bigbluebutton.org/.
+#
+# Copyright (c) 2022 BigBlueButton Inc. and by respective authors (see below).
+#
+# This program is free software; you can redistribute it and/or modify it under the
+# terms of the GNU Lesser General Public License as published by the Free Software
+# Foundation; either version 3.0 of the License, or (at your option) any later
+# version.
+#
+# Greenlight is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License along
+# with Greenlight; if not, see <http://www.gnu.org/licenses/>.
+
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class ServerTagsController < ApiController
+      # GET /api/v1/server_tags/:friendly_id
+      # Returns a list of all allowed tags&names for the room's owner
+      def show
+        room = Room.find_by!(friendly_id: params[:friendly_id])
+        tag_names = Rails.configuration.server_tag_names
+        tag_roles = Rails.configuration.server_tag_roles
```

Comment:
I think it may be better to add a guard clause here to handle the case where server tags is not enabled

```
tag_names = Rails.configuration.server_tag_names
tag_roles = Rails.configuration.server_tag_roles

return render_data data: {}, status: :ok if tag_names.blank?
```

Reply:
At least according to my testing, right now exactly the same happens when the SERVER_TAG_NAMES env is unset or empty.

Reply:
Anyway addressed in https://github.com/bigbluebutton/greenlight/pull/5790/commits/b3c2b47b40fda56b101c513ff40f7c342558341b . The benefit is indeed that a useless Room.find is avoided.